README.md

```diff
@@ -1 +1,41 @@
-# Enumerable
\ No newline at end of file
+# Enumerable
+This is project of creating customized Enurables  using Ruby.
+
+## Built With
+
+- Ruby 
+
+## Author 1
```

Comment:
- [ ] Let's once again look at the template from Microverse which we can use to draft a professional readme file. Your readme file is missing a "getting started" section. Just imagine someone wants to use your project, what are the commands required to run the project on their local machines?